docs/quick-start.md

```diff
@@ -185,20 +163,12 @@ Let's break that down:
   </tr>
 </table>
 
-As described above, the `widget` property specifies a built-in or custom UI widget for a given field. The first field in the example, `layout`, uses a `hidden` widget. This widget will not show in the editor UI, but will be saved with the `default` value (assuming it's been set) in the document front matter. The rest of the widgets work as follows:
-
-Widget | UI | Data Type
---- | --- | ---
-`string` | text input | string
-`datetime` | date picker widget | ISO date string
-`image` | file picker widget with drag-and-drop | file path saved as string, image uploaded to media folder
-`number` | text input with `+` and `-` buttons | number
-`markdown` | rich text editor with raw option | markdown-formatted string
+As described above, the `widget` property specifies a built-in or custom UI widget for a given field. When a content editor enters a value into a widget, that value will be saved in the document front matter as the value for the `name` specified for that field. A full listing of available widgets can be found in the [Widgets doc](/docs/widgets).
 
 Based on this example, you can go through the post types in your site and add the appropriate settings to your `config.yml` file. Each post type should be listed as a separate node under the `collections` field.
 
 ### Filter
```

Comment:
I don't know how this made it back in, would you mind removing? Filtering shouldn't be in the quick start guide.

Reply:
I don't think it ever left. And yeah, totally agree it doesn't belong here, but it's not documented anywhere else. I don't see a better place for it in the existing docs, so I suggest we leave it here for the next short while until we have a dedicated config doc.